test/integration/boundingbox_camera_plugin.cc

```diff
@@ -0,0 +1,474 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include <gtest/gtest.h>
+
+#include <ignition/common/Filesystem.hh>
+#include <ignition/sensors/Manager.hh>
+#include <ignition/sensors/BoundingBoxCameraSensor.hh>
+
+#include <ignition/rendering/RenderEngine.hh>
+#include <ignition/rendering/RenderingIface.hh>
+#include <ignition/rendering/Scene.hh>
+#include <ignition/rendering/BoundingBoxCamera.hh>
+#include <ignition/msgs.hh>
+
+#include "test_config.h"  // NOLINT(build/include)
+#include "TransportTestTools.hh"
+
+using namespace ignition;
+using namespace rendering;
```

Comment:
```suggestion
```
We shouldn't be using the `rendering` namespace in this repository since it's `ign-sensors`, not `ign-rendering`